.gitignore

```diff
@@ -0,0 +1 @@
+test.md
```

Comment:
- [ ] Ensure to add your `node_modules` directory under the `.gitignore`.

Reply:
Thank you

Reply:
- [x] Node module and test.md have been added to `.gitignore`